.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Run Linters
+on:
+  - push
```

Comment:
I think we need:
```
on:
  - push
  - pull_request
```
With that we can remove the lints from azure.

the m
  - pull_request

Reply:
done.